src/stories/Library/material-grid-related/material-grid-related.scss

```diff
@@ -0,0 +1,78 @@
+.material-grid-related {
+  margin-bottom: 1rem;
+}
+
+.material-grid-related__header {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin-bottom: 20px;
+  @include layout-container($layout__max-width--large);
+
+  @include media-query__small {
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 30px;
+  }
+}
+
+.material-grid-related__title {
+  @include typography($typo__h2);
+}
+
+.material-grid-related__dropdown {
+  display: inline-flex;
+  align-items: center;
+
+  @include media-query__small {
+    display: none;
+  }
+}
+
+.material-grid-related__filter-inline {
+  display: none;
+  margin-top: auto;
+  gap: $s-md;
+
+  @include media-query__small {
+    display: flex;
+  }
+}
+
+.material-grid-related__filter-button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  font-weight: $font__weight--normal;
+  color: inherit;
+  position: relative;
+  display: inline-flex;
+  align-items: flex-start;
+  gap: 8px;
+}
+
+.material-grid-related__filter-text {
+  position: relative;
+  display: inline-block;
```

Comment:
Please consider adding `text-transform: uppercase;` here.

As I see it the casing of the filers are part of the design, not a translation.